mds/models.py

```diff
@@ -237,11 +238,10 @@ class Area(models.Model):
     label = UnboundedCharField(default="", blank=True, db_index=True)
     polygons = models.ManyToManyField(Polygon, blank=True, related_name="areas")
     providers = models.ManyToManyField(Provider, blank=True, related_name="areas")
-    color = models.CharField(
-        max_length=8, default="#FFFFFF", help_text="hexa representation"
-    )
+    color = UnboundedCharField(default="#FFFFFF", help_text="hexa representation")
```

Comment:
A bit too eager to replace all CharFields?

Reply:
as stated in https://www.postgresql.org/docs/11/datatype-character.html,  in fact `text` is a faster type anywhere, and implementation details state that `char` and `char var` types in Postgres are just additional layers (length check, blank write) over the same underlying generic implementation. So I find this KISS to use `text` everywhere.